inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/convolution.hpp

```diff
@@ -43,6 +44,15 @@ class ConvolutionLayerTest : public testing::WithParamInterface<convLayerTestPar
 
 protected:
     void SetUp() override;
+    std::shared_ptr<ngraph::Function> makeConvolution(const std::string& name = "");
```

Comment:
I write the comment about such construstion in the past. I think it is not required to be placed to separated function from 'SetUp'. I would like to return such code to the `SetUp()` with using `set_friendly name` function

Reply:
Your comment was confused to use ngraph::clone_function() and it will be returned to the original function.

Reply:
![image](https://user-images.githubusercontent.com/22008923/135159109-566cde5b-8b19-415e-8418-37d3cfc62074.png)
I want to see such code :) Only without prints

Reply:
Yeah, I knew it, even though not exactly, and functionRefs is used more than one or two hundreds. So I skipped the name. (I'm not used to scripting.)

Reply:
We can skip set_friendly_name for all cases with the exception of `softmax` and `convolution` cases